doctr/models/recognition/transformer/tensorflow.py

```diff
@@ -239,6 +255,8 @@ def call(
         x *= tf.math.sqrt(tf.cast(self.d_model, tf.float32))
         x += self.pos_encoding[:, :seq_len, :]
 
+        x = self.dropout(x, **kwargs)
```

Comment:
Mmmh, we already have dropout in the `DecoderLayer`, are you positive we're supposed to put one here?

Reply:
https://www.tensorflow.org/text/tutorials/transformer#decoder Here they put another one here, but we can remove it

Reply:
In pytorch it is also used here after the `encode_sequences`: https://pytorch.org/tutorials/beginner/transformer_tutorial.html#define-the-model